server/controller/taskController.js

```diff
@@ -2,6 +2,7 @@ const taskService = require('../service/taskService');
 
 const taskController = {
     getTasks: (req, res) => {
+        console.log(req.query)
```

Comment:
When you are making PR, remove all unecessary code, like temporary comments, console.logs. We want to have clear log output on client and server. Please remove all unecessary code.

Reply:
Removed